File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/multi_import_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

multi_import_a;
multi_import_b;

x = multi_import_b.XXX();
if (x.testx() != 0)
  error
endif

y = multi_import_b.YYY();
if (y.testx() != 0)
  error
endif
if (y.testy() != 1)
  error
endif

z = multi_import_a.ZZZ();
if (z.testx() != 0)
  error
endif
if (z.testz() != 2)
  error
endif

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/multi_import_runme.go
package main

import "multi_import_a"
import "multi_import_b"

func main() {
	x := multi_import_b.NewXXX()
	if x.Testx() != 0 {
		panic(0)
	}

	y := multi_import_b.NewYYY()
	if y.Testx() != 0 {
		panic(0)
	}
	if y.Testy() != 1 {
		panic(0)
	}

	z := multi_import_a.NewZZZ()
	if z.Testx() != 0 {
		println("z.Testx", z.Testx(), z.Testz())
		panic(0)
	}
	if z.Testz() != 2 {
		println("z.Testz", z.Testz())
		panic(0)
	}
}
